components/automate-deployment/pkg/server/api_token.go

```diff
@@ -63,8 +64,9 @@ func generateAdminToken(ctx context.Context,
 	authzV2Client := authz_v2.NewPoliciesClient(authzConnection)
 
 	response, err := authnClient.CreateToken(ctx, &authn.CreateTokenReq{
```

Comment:
💭 Was this call idempotent before? If so, is it still? I don't know from the top of my head where this method is called, but it might be good to double-check.

Reply:
Naw, before we were just using authz-service to generate the UUID for us.